arm-sql/2014-04-01/swagger/dataMasking.json

```diff
@@ -0,0 +1,460 @@
+                ],
+                "operationId": "DataMasking_ListRules",
+                "description": "Returns a list of Azure SQL database data masking rules.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameter"
+                    },
+                    {
+                        "name": "databaseName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL Database for which the data masking rules applies."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/DataMaskingRuleListResult"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        }
+    },
+    "definitions": {
+        "DataMaskingPolicyProperties": {
+            "properties": {
+                "dataMaskingState": {
+                    "type": "string",
+                    "description": "Gets or sets the state of the data masking policy."
+                },
+                "exemptPrincipals": {
+                    "type": "string",
+                    "description": "Gets or sets the list of the exempt principals"
+                }
+            },
+            "required": [
+                "dataMaskingState",
+                "exemptPrincipals"
+            ],
+            "description": "Represents the properties of an Azure SQL Database data masking policy."
+        },
+        "DataMaskingPolicy": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/DataMaskingPolicyProperties",
+                    "description": "Represents the properties of the resource."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/SubResource"
+                }
+            ],
+            "description": "Represents an Azure SQL Database data masking policy."
+        },
+        "DataMaskingRuleProperties": {
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "Gets or sets the rule Id."
+                },
+                "ruleState": {
```

Comment:
@yoavrubin 

---
In reply to: [80149671](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80149671) [](ancestors = 80149671)